bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/ModelConverters.scala

```diff
@@ -194,7 +198,13 @@ object ModelConverters {
         case t:HmsDegTarget => Some(t.getRedshift)
         case _              => None
       }
-      SiderealTarget(name, coordinates, pm, z, px, mags)
+      SiderealTarget.empty.copy(
```

Comment:
Yeah, I am wondering, if we don't loose the `SpectralDistribution` and `SpatialProfile` if we convert back and forth between the two? I am not sure if it matters right now, since ITC currently takes those values from the old model (ITarget).